examples/kotlin-diktat/fix/smoke/save.toml

```diff
@@ -0,0 +1,6 @@
+[general]
+tags = "smoke"
```

Comment:
```suggestion
tags = smoke
```
For now?

Reply:
Yeah, thanks, I'll remove all quotes for consistency.